pkg/asset/installconfig/openstack/validate.go

```diff
@@ -39,3 +39,11 @@ func Validate(ic *types.InstallConfig) error {
 
 	return allErrs.ToAggregate()
 }
+
+// ValidateForProvisioning validates that the install config is valid for provisioning the cluster.
+func ValidateForProvisioning(ic *types.InstallConfig) error {
+	if ic.ControlPlane.Replicas != nil && *ic.ControlPlane.Replicas != 3 {
```

Comment:
I think it would be more appropriate to put the check in the [type validation](https://github.com/openshift/installer/blob/master/pkg/types/validation/installconfig.go#L369). The validation where you have it is meant to be for validation that requires access to the platform provider.

Reply:
I thought about doing that, but I ended up putting it here because I didn't want UPI installs to also be caught by this check. My intention is to guard against the very narrow case of an IPI install on OpenStack.

Reply:
OK. I'll buy that.